language/move-lang/src/expansion/translate.rs

```diff
@@ -69,8 +70,9 @@ impl Context {
         }
     }
 
-    pub fn clear_aliases(&mut self) {
-        self.aliases = AliasMap::new();
+    pub fn clear_aliases(&mut self) -> BTreeSet<Name> {
```

Comment:
Can you add a comment to explain the return value? It looks like it is the set of aliases that were ever used, but it would be good to make that clear.